### PR TITLE
Fix deleting projects after asset UI update.

### PR DESCRIPTION
**Pull request by mbarkley**

### CODE REVIEW

**Comment by mbarkley** [comment]
Related:
https://github.com/errai/errai/pull/333
https://github.com/kiegroup/appformer/pull/223
https://github.com/kiegroup/kie-wb-common/pull/1475

**Comment by mbarkley** [comment]
@adrielparedes This PR set contains a fix for an NPE when deleting projects that I think was introduced by the asset UI changes. Would you mind reviewing?

File: uberfire-project/uberfire-project-backend/src/test/java/org/guvnor/common/services/project/backend/server/AbstractModuleServiceTest.java [hunk 1]
```diff
@@ -134,4 +142,43 @@ public void testRenameEventFiredBeforeDeleteEvent() {
...
+        when(repo.getSpace()).thenReturn(space);
+
+        abstractProjectService.delete(path, "");
+
+        verify(repoService).removeRepository(eq(space), eq(alias));
+        verify(ioService, times(0)).delete(any(), any());
+    }
+
+    @Test
+    public void testUseIOServiceToDeleteRootModule() {
```
Comment (paulovmr) [inline uberfire-project/uberfire-project-backend/src/test/java/org/guvnor/common/services/project/backend/server/AbstractModuleServiceTest.java]: I think the name should be `testUseIOServiceToDeleteSubModule` (or something like that) instead.

Comment (mbarkley) [reply uberfire-project/uberfire-project-backend/src/test/java/org/guvnor/common/services/project/backend/server/AbstractModuleServiceTest.java]: Oh yeah, good catch.

**Comment by mbarkley** [comment]
@paulovmr Updated PR with your suggestion.

**Pull request merged**